data/sitesFR.json

```diff
@@ -41,6 +41,25 @@
     "destination_search_path": "/index.php",
     "destination_content_path": "/"
   },
+  {
+    "id": "fr-evangelion",
+    "origins_label": "Evangelion Fandom Wiki",
+    "origins": [
+      {
+        "origin": "Evangelion Fandom Wiki",
+        "origin_base_url": "evangelion.fandom.com/fr",
+        "origin_content_path": "/wiki/",
+        "origin_main_page": "Wiki_Evangelion"
+      }
+    ],
+    "destination": "Shin Wiki Evangelion",
+    "destination_base_url": "shinwikievangelionfr.miraheze.org",
+    "destination_platform": "mediawiki",
+    "destination_icon": "shinwikievagelion.png",
+    "destination_main_page": "Wiki_Evangelion",
+    "destination_search_path": "/index.php",
```

Comment:
Missing `destination_content_path` (also looks like `fr-lethalcompany` is missing a `destination_content_path`, if you could add that while you're in this file) - thank you!